pkg/ai/openai.go

```diff
@@ -27,8 +28,12 @@ type OpenAIClient struct {
 	model    string
 }
 
-func (c *OpenAIClient) Configure(token string, model string, language string) error {
-	client := openai.NewClient(token)
+func (c *OpenAIClient) Configure(token string, model string, language string, baseURL string) error {
```

Comment:
It feels like we could give llama its own struct rather than modifying the OpenAI one, what do you think?

Reply:
I'm definitely ok with it, however it will look like mostly the same - behind the scenes it's going still to use the OpenAI client. I'll wrap my changes in a separate struct so we can compare and pick the one we like!

Reply:
done!